scripts/wsl2_ai_tuning.py

```
@@ -0,0 +1,401 @@
+            else:
+                print("  [+] Open file limits already set in /etc/security/limits.conf")
+        except Exception as e:
+            print(f"  [-] Failed modifying limits.conf: {e}")
+
+    # 3. /etc/wsl.conf
+    wsl_conf = Path("/etc/wsl.conf")
+    if os.geteuid() == 0:
+        try:
+            wsl_conf.write_text(generate_wsl_conf())
+            print("  [+] /etc/wsl.conf updated with systemd, GPU, and automount settings.")
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)